R/module-statmodel-server.R

```diff
@@ -909,6 +908,7 @@ statmodelServer <- function(input, output, session,parent_session, loadpage_inpu
   return(
     list(
       input = input,
+      significant = SignificantProteins,
```

Comment:
I think this is where we had talked about using the proteins being already filtered.  I think this may not be necessary and you can use the dataComparison object as the list of proteins with their p-values